### ISSUE TITLE
BUG: sklearn version must be >= 0.18.1

### ISSUE THREAD

**Issue by kcrum** [civisanalytics/civisml-extensions#3]
The `HyperbandSearchCV` class depends on the `MaskedArray` class, which was added to `sklearn.utils.fixes` in version 0.18.1. Attempts to import `civismlext` fail when using sklearn v0.18. 

Here's what that looks like:

 ```python
In [1]: from civismlext import NonNegativeLinearRegression
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-1-86087081ae3b> in <module>()
----> 1 from civismlext import NonNegativeLinearRegression

/Users/kcrum/miniconda3/envs/sandbox/lib/python3.5/site-packages/civismlext/__init__.py in <module>()
      2 from civismlext.stacking import StackedClassifier  # NOQA
      3 from civismlext.nonnegative import NonNegativeLinearRegression  # NOQA
----> 4 from civismlext.hyperband import HyperbandSearchCV  # NOQA
      5 from civismlext.preprocessing import DataFrameETL  # NOQA

/Users/kcrum/miniconda3/envs/sandbox/lib/python3.5/site-packages/civismlext/hyperband.py in <module>()
     18 
     19 from sklearn.externals.joblib import Parallel, delayed
---> 20 from sklearn.utils.fixes import MaskedArray
     21 from sklearn.utils.validation import indexable
     22 from sklearn.metrics.scorer import check_scoring

ImportError: cannot import name 'MaskedArray'
In [2]: 
In [2]: from sklearn.utils.fixes import MaskedArray
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-2-f7d936768fbc> in <module>()
----> 1 from sklearn.utils.fixes import MaskedArray

ImportError: cannot import name 'MaskedArray'
In [3]: import sklearn
In [4]: sklearn.__version__
Out[4]: '0.18'
```

**Comment by beckermr** [comment]
Hmmmm. Usually sklearn removes these fixes as things get fixed upstream. We should check if 0.19 has this still. If it does not, then we should pull this from a later version of numpy.

**Comment by kcrum** [comment]
It seems the `MaskedArray` fix is still in 0.19.

**Comment by beckermr** [comment]
Yep looks like they use it if you have numpy < 1.12.

**Comment by kcrum** [comment]
Resolved by PR #4
